Lib/ast.py

```diff
@@ -643,6 +643,14 @@ def main():
                              'column offsets')
     parser.add_argument('-i', '--indent', type=int, default=3,
                         help='indentation of nodes (number of spaces)')
+    parser.add_argument('--feature-version',
+                        type=str, default=None, metavar='VERSION',
+                        help='python version in the format 3.x (e.g., 3.10)')
+    parser.add_argument('-o', '--optimize',
```

Comment:
I would prefer -O instead, so to mimick the -OO of the interpreter (this is *not* the same kind of optimization IIRC)